doc/classes/TileSet.xml

```diff
@@ -36,12 +36,66 @@
 			<description>
 			</description>
 		</method>
+		<method name="autotile_clear_bitmask_map">
+			<return type="void">
+			</return>
+			<argument index="0" name="id" type="int">
+			</argument>
+			<description>
+				Clears all bitmask info of the autotile.
+			</description>
+		</method>
+		<method name="autotile_get_bitmask">
+			<return type="int" enum="TileSet.AutotileBindings">
+			</return>
+			<argument index="0" name="id" type="int">
+			</argument>
+			<argument index="1" name="coord" type="Vector2">
+			</argument>
+			<description>
+				Returns the bitmask of the subtile from an autotile given its coordinates.
+				The value is the sum of the values in [enum AutotileBindings] present in the subtile (e. g. a value of 5 means the bitmask has bindings in both the top left and top right).
```

Comment:
The `[enum]` syntax is a bit limited right now in the in-editor help, you need to specify the class otherwise it will look into `@GlobalScope`. So this should be `[enum TileSet.AutotileBindings]` (same below for `[enum TileSet.BitmaskMode]`).